.github/ISSUE_TEMPLATE/request-for-change.md

```diff
@@ -0,0 +1,13 @@
+---
+name: DCR Request for change
+about: Tell us about a feature that you're working on
+title: "[RFC]"
+labels: RFC, dotcom-rendering
+assignees: ''
+
+---
+
+The Dotcom team would like to help you with your proposed change. Please provide us with a brief and concise explanation for what you're looking to do we will try to make the process easier with early feedback.
```

Comment:
Is it worth indicating the type of change an RFC is necessary for? In principal this seems like a great idea, but it would be good to lay down the scale of change this applies for. 

There are also other teams that are part responsible for some code in dotcom-rendering (thinking commercial / T&C etc.) Would BAU changes to code those teams are concerned with require an RFC, i'd guess not but worth laying it out?

Reply:
This is a good point. I don't think anyone wants to be asked to jump through hoops when making changes to their 'own' code so we need to be clear about when this might or might now apply.

I think in reality what we will end up on is asking people to use their own best judgement. We're a clever bunch here at the Guardian and, more than that, we're very good at working autonomously and making the right choices and so maybe am optional approach would work. But equally, there are benefits to automation taking over and reducing the amount of thinking engineers need to do. What do you think?

Reply:
> I think in reality what we will end up on is asking people to use their own best judgement.

This makes sense, but for new folk it's probably worth being clear (perhaps with examples) in the template about when an RFC would be appreciated.

Reply:
👍  I've added more information to the description to make it clearer about when this RFC should apply